invenio_communities/assets/semantic-ui/js/invenio_communities/community/communitiesCarousel/CarouselItem.js

```diff
@@ -44,14 +44,34 @@ class CarouselItem extends Component {
                   content={i18next.t("Browse")}
                 />
                 {showUploadBtn && (
-                  <Button
-                    size="mini"
-                    icon="upload"
-                    labelPosition="left"
-                    positive
-                    href={`/uploads/new?community=${community.slug}`}
-                    content={i18next.t("New upload")}
-                  />
+                  <>
+                    <Button
+                      size="mini"
+                      icon="upload"
+                      labelPosition="left"
+                      positive
+                      href={`/uploads/new?community=${community.slug}`}
+                      content={i18next.t("New upload")}
+                      disabled={!canSubmitRecord}
+                    />
+                    {!canSubmitRecord && (
+                      <InvenioPopup
+                        popupId="community-inclusion-info-popup"
+                        size="small"
+                        trigger={
+                          <Icon
+                            className="mb-5"
+                            color="grey"
+                            name="question circle outline"
+                          />
+                        }
+                        ariaLabel={i18next.t("Community submission policy information")}
```

Comment:
I think after @ptamarit presentation, this should depict the actual information so to be useful right? 
```suggestion
                        ariaLabel={i18next.t("Submission to this community is only allowed to community members.")}
```